### DISCUSSION TITLE
Enable lazy by default in scratchpads

### DISCUSSION THREAD

**Discussion by fdev31** [hyprland-community/pyprland#47] [Polls]
# Scratchpads : lazy

A lot of users use the lazy option, the default option is more designed for slow computers and not so standard apps.

Users using web apps in scratchpads usually resuire `lazy` to be `true`.
Todays behavior (lazy=false) means scratchpads are running on session startup, which can also degrade the first seconds of it.

**Comment by fdev31** [comment]
That's not helping guys! ;)
Do you think about a way to set it automatically or maybe get rid of this feature ?
Just some random idea to start the discussion:

- implement randomization of the startup of the apps (using a priority?) when lazy is false
- keeping the lazy for users not wanting to run the app at all if they don't use it

This would make more "weird apps" behave as expected out of the box... but maybe you are not using lazy for such apps?

In short, what are you using this option for? I would try to get rid of it for "most users".

**Reply by axelKeizoStahl** [reply]
I think default lazy makes most sense. 
For example, when I have discord open on my computer, my phone does not give notifications. I would want to not get notifications on my phone if I am actually using it on my computer (when I start its scratchpad).
More generally, I think it makes sense (generally) to say the user is not using the app if they haven't opened it, and if they aren't using it, why should it be running.

**Comment by fdev31** [comment]
There is now a significant difference anyways, approved.
